src/monaco-editor/monaco-loader.ts

```diff
@@ -15,6 +17,22 @@ export function loadMonaco(): Promise<Monaco> {
 				validate: false, //Turn off CSS built-in validation.
 			});
 
+			monaco.languages.json.jsonDefaults.setDiagnosticsOptions({
+				validate: true,
+				enableSchemaRequest: false, // TODO: When switching a remote schema, enable it.
+				schemas: [
+					{
+						uri: 'https://json.schemastore.org/stylelintrc',
+						fileMatch: ['.stylelintrc.json'],
```

Comment:
Should these line be commented?

Reply:
For `uri`, maybe we can comment it out because it seems to have no effect.

But, for `fileMatch`, we cannot comment it out because of the side-effect on other JSON files like `package.json`.

Reply:
I find it misleading to leave the `uri` if we are not actually using it.

Reply:
Make sense. Since `uri` is required, I've changed it to one in `schemaStylelintrc` with b271868.